src/random_test.cpp

```diff
@@ -1608,28 +1609,92 @@ void Table::DeleteRandomRow(Thd1 *thd) {
     }
   }
 
-  std::string sql = "DELETE FROM " + name_ + " WHERE " +
-                    columns_->at(where)->name_ + "=" +
+  std::string sql1 = "DELETE FROM " + name_ + " WHERE " +
```

Comment:
rather than defining 4 variables at run time. 

use single variables. and based on probability assign it values.

like if (prob == 1)
 sql = ""
else if (prob == 2) 
sql = ""

Reply:
Fixed